akka-docs/src/main/paradox/typed/guide/actors-motivation.md

```diff
@@ -0,0 +1,159 @@
+# Why modern systems need a new programming model
```

Comment:
Followup (maybe?): This can be moved to a more reference focussed section of the docs and linked to from both?

Reply:
I agree, I think we already have some link indicating that this section can be skipped